website/blog/2022-01-11-aluminium.md

```diff
@@ -0,0 +1,585 @@
+---
+author: Tomasz Godzik
+title: Metals v0.11.0 - Aluminium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.11.0! Due to the number of
+changes in this release we decided to introduce one more minor version before
+Metals 1.0.0, which is planned to come this year.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">313</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">129</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">11</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">10</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/<num>?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Add CFR class file viewer.
+- [Scala 3] Show type on hover.
+- Simplify usage of sbt server.
+- Better support for mill BSP.
+- Extract value code action.
+- Add command to run current file.
+- Support test explorer API.
+- Add go to definition via links for synthetic decorations.
+- Basic Java support.
+- Add support for Scala 2.13.8
+
+## Add CFR class file viewer
+
+TODO
+
+## [Scala 3] Show type on hover
+
+TODO
+
+## Simplify usage of sbt server
+
+TODO
+
+## Better support for mill BSP
+
+TODO
```

Comment:
Recently [Mill](https://github.com/com-lihaoyi/mill) has complete revamped their BSP implementation allowing for a much better experience for Metals users. Just like sbt, you can now use the `metals.bsp-switch` command to use Mill as your BSP server (needs at least Mill 0.10.0-M4). Keep in mine that the BSP support in Mill is still a work-in-progress. If you are interested in Mill BSP and you're a Metals user, feel free to include your input [here](https://github.com/com-lihaoyi/mill/issues/1546) as they are requesting feedback on improving the experience.